server/http.go

```diff
@@ -403,6 +405,79 @@ func (p *Plugin) handleStartMeeting(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (p *Plugin) handleMobileStart(w http.ResponseWriter, r *http.Request) {
```

Comment:
I'm not familiar with the Zoom plugin specifically, but I'm guessing the idea here is that this would be the same logic that it currently does to start the meeting but only possibly returning the result action.

Reply:
Exactly. This was a fast implementation, but the good implementation would be to have function to handle the start that returns success or error, and form the response depending on whether it is the webapp header button, the command or the plugin integration.